pkg/collector/corechecks/network-devices/cisco-sdwan/client/request.go

```diff
@@ -120,14 +119,14 @@ func getMoreEntries[T Content](client *Client, endpoint string, params map[strin
 
 		log.Tracef("Getting page %d from endpoint %s", page+1+1, endpoint)
 		// Update the params to get next API page
-		err := updatePaginationParams(currentPageInfo, params)
+		nextParams, err := getNextPaginationParams(currentPageInfo, client.maxCount)
 		if err != nil {
 			return nil, err
 		}
 		log.Tracef("Pagination params for page %d from endpoint %s : %v", page+1+1, endpoint, params)
```

Comment:
Should we log `nextParams` instead?

Reply:
Also it doesn't seem like we use `params` anywhere, should we remove this parameter from the function arguments? (and to any code that calls it)

Reply:
🤦‍♂️🤦‍♂️🤦‍♂️ good catch thanks!